packages/pkl-ts-parser/antlr.d.ts

```diff
@@ -0,0 +1,3 @@
+export * from 'antlr4ng';
+
+export {};
```

Comment:
`import * as antlr from 'antlr4ng';`
`export { antlr };`

This code does the same as you wrote above right?

Reply:
no. `export { antlr };` exports only one key. but `export * from 'antlr4ng';` exports every key in `antlr4ng`